esp-hal/src/uart.rs

```diff
@@ -1133,8 +1132,8 @@ where
         sync_regs(register_block);
     }
 
-    /// Write a byte out over the UART
-    pub fn write_byte(&mut self, word: u8) -> nb::Result<(), Error> {
+    // Write a byte out over the UART
+    fn write_byte(&mut self, word: u8) -> nb::Result<(), Error> {
```

Comment:
We could also delete these and just replace the instances where its used with the equivalent oneline, no hard opinion though.

Reply:
I left this so it is "consistent" with `read_byte` and I marked this one as private in UartRx.